src/config.rs

```diff
@@ -162,6 +163,15 @@ impl Config {
                 if let Ok(env_token) = env::var(AUTH_TOKEN_ENV) {
                     tokens.insert(env_token);
                 }
+                if let Ok(env_path) = env::var(AUTH_FILE_ENV) {
+                    match read_to_string(&env_path) {
+                        Ok(s) => push_token_file(s, &mut tokens),
```

Comment:
No need for a function call.

```suggestion
                        Ok(s) => {
                            s.lines().filter(|l| !l.trim().is_empty()).for_each(|l| {
                                tokens.insert(l.to_string());
                            });
                        }
```